src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -783,6 +783,27 @@ trait GoldTransforms extends SparkSessionWrapper {
     df.select(sparkExecutionCols: _*)
   }
 
+  private def cleanseDesc(colName: String): Column = {
+    trim(
+      when(col(colName).like("%<br/>%"), regexp_replace(col(colName), "<br/>", " "))
+        .when(col(colName).like("%\n%"), regexp_replace(col(colName), "\\n", " "))
+        .otherwise(col(colName))
```

Comment:
I mainly did it to make it easier to read. I don't think it will actually pass through the data twice, it's a case statement it will either match case a or case b or else. Nonetheless, after thinking about it I think I like your idea better. Please review the updated logic and make sure they look correct.